sdk/cognitivelanguage/ai-language-text/test/public/analyzeBatch.spec.ts

```diff
@@ -688,7 +688,7 @@ matrix([["APIKey", "AAD"]] as const, async (authMethod: AuthMethod) => {
         });
 
         // see https://github.com/Azure/azure-sdk-for-js/issues/23900
```

Comment:
```suggestion
```

The issue link can be removed as well. Could you please update the issue accordingly after the PR is merged?